src/fal/cli/cli.py

```diff
@@ -16,6 +16,7 @@
 def cli(argv: List[str] = sys.argv):
     # Wrapper to be able to shutdown telemetry afterwards
     try:
+        telemetry.persist_hashed_project_name(argv[1:])
```

Comment:
I think this would not be called for using fal as a library, right?

```
from fal import FalDbt

faldbt = FalDbt(...)
faldbt.source(...) # no project set
```

Should we set this in `FalDbt.__init__` and use the `config` object which I think already has a hashing solution?

Reply:
Good catch, @chamini2. My only concern would then be the telemetry call on line 26. What do you think about it?

Reply:
I wonder if we can _spare_ the project identifier for that call and we can "tie it" to the rest of the call by giving an `invocation_id` (do we have one of those?)

Reply:
Ok, so, PR has been updated with limited scope as discussed.